packages/curve-ui-kit/src/widgets/Legal/components/tabs/Terms.tsx

```diff
@@ -0,0 +1,338 @@
+import Link from '@mui/material/Link'
+import Typography from '@mui/material/Typography'
+import { CurveNetworkId, LlamaNetworkId } from '@ui-kit/features/connect-wallet/lib/types'
+import { t } from '@ui-kit/lib/i18n'
+import { AppName, getInternalUrl, PAGE_LEGAL } from '@ui-kit/shared/routes'
+import { RouterLink } from '@ui-kit/shared/ui/RouterLink'
+import { pushSearchParams } from '@ui-kit/utils/urls'
+import { List } from '../general/List'
+import { Section, Header, Title, Paragraph, Bold, SubTitle } from '../general/Section'
+
+export const Terms = ({ currentApp, network }: { currentApp: AppName; network: CurveNetworkId | LlamaNetworkId }) => (
+  <>
+    <Header>{t`Preamble`}</Header>
+    <Section>
+      <Paragraph>
+        {t`These terms and conditions (`}
```

Comment:
FYI it's not really important rn as our translation is turned off, but usually it's better to use the [`Trans` placeholder component](https://github.com/curvefi/curve-frontend/blob/de2dace158df548344ccf78e820d76c9b884d958/packages/curve-ui-kit/src/lib/i18n.ts#L16) instead of breaking a sentence into multiple `t` calls